include/ua_plugin_nodestore.h

```diff
@@ -177,12 +180,23 @@ typedef enum {
         UA_DataSource dataSource;                                       \
     } value;
 
+#ifdef UA_ENABLE_HISTORIZING
+typedef struct {
+    UA_UInt64 historizingCallbackId;
+    UA_NodeId * nodeId;
+} UA_HistorizingInternal;
+#endif
+
 typedef struct {
     UA_NODE_BASEATTRIBUTES
     UA_NODE_VARIABLEATTRIBUTES
     UA_Byte accessLevel;
     UA_Double minimumSamplingInterval;
-    UA_Boolean historizing; /* currently unsupported */
+    UA_Boolean historizing;
+#ifdef UA_ENABLE_HISTORIZING
```

Comment:
I would make the HDA plugin global for the entire server.
Storing the settings in the node itself can be problematic.

When we make a copy of the node, we don't know whether this is to make a new node or to replace the existing node with a modified copy.